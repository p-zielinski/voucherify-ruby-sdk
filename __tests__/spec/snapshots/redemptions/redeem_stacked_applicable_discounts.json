{
  "redemptions": [
    {
      "id": "r_0f183d6b5f861bd967",
      "date": "2024-07-22T10:39:11.230Z",
      "order": {
        "id": "ord_0f183d6b59061bd960",
        "source_id": null,
        "status": "PAID",
        "customer_id": null,
        "referrer_id": null,
        "amount": 20000,
        "applied_discount_amount": 0,
        "total_applied_discount_amount": 0,
        "total_amount": 20000,
        "metadata": {},
        "created_at": "2024-07-22T10:39:11.218Z",
        "object": "order"
      },
      "result": "SUCCESS",
      "status": "SUCCEEDED",
      "voucher": {
        "id": "v_z3zt00YdjMmCe3yUW0pKYO26joOyh7K0",
        "code": "Q7YFDWIR",
        "discount": {
          "type": "AMOUNT",
          "amount_off": 1000
        },
        "type": "DISCOUNT_VOUCHER",
        "campaign": "WCMdmKCMfh",
        "campaign_id": "camp_5mAvrP5okAssEE9TsEowFPfb",
        "is_referral_code": false,
        "holder_id": "cust_pJtSKTdFWGVZs53iSkxmHYn0",
        "category_id": null,
        "active": true,
        "created_at": "2024-07-22T10:39:09.197Z",
        "updated_at": "2024-07-22T10:39:11.227Z",
        "redemption": {
          "quantity": null,
          "redeemed_quantity": 1
        },
        "start_date": null,
        "expiration_date": null,
        "metadata": {},
        "object": "voucher"
      },
      "metadata": null,
      "object": "redemption"
    }
  ],
  "order": {
    "id": "ord_0f183d6b59061bd960",
    "source_id": null,
    "created_at": "2024-07-22T10:39:11.218Z",
    "updated_at": null,
    "status": "PAID",
    "amount": 20000,
    "total_amount": 20000,
    "applied_discount_amount": 0,
    "total_applied_discount_amount": 0,
    "metadata": {},
    "customer_id": null,
    "referrer_id": null,
    "object": "order",
    "redemptions": {
      "r_0f183d6b5f861bd967": {
        "date": "2024-07-22T10:39:11.230Z",
        "related_object_type": "voucher",
        "related_object_id": "v_z3zt00YdjMmCe3yUW0pKYO26joOyh7K0",
        "related_object_parent_id": "camp_5mAvrP5okAssEE9TsEowFPfb"
      }
    }
  },
  "inapplicable_redeemables": [],
  "skipped_redeemables": []
}
