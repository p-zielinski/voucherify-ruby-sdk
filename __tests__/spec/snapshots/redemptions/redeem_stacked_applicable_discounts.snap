{:redemptions=>[{:id=>"r_0f180c999a51028c1e", :object=>"redemption", :date=>2024-07-22 07:05:53.513 UTC, :metadata=>nil, :result=>"SUCCESS", :status=>"SUCCEEDED", :order=>{:id=>"ord_0f180c994ed1028c17", :source_id=>nil, :status=>"PAID", :amount=>20000, :total_amount=>20000, :applied_discount_amount=>0, :total_applied_discount_amount=>0, :metadata=>{}, :customer_id=>nil, :referrer_id=>nil, :object=>"order"}, :voucher=>{:id=>"v_B5u2zEUX75M0nGADR59zl5Ikd4qnxxeT", :code=>"AuNu9SnW", :campaign=>"7fqeLn4hzy", :campaign_id=>"camp_1s1GPkPitiPpG6QJjK744cJR", :category_id=>nil, :type=>"DISCOUNT_VOUCHER", :discount=>{:type=>"AMOUNT", :amount_off=>1000.0}, :start_date=>nil, :expiration_date=>nil, :active=>true, :metadata=>{}, :is_referral_code=>false, :updated_at=>2024-07-22 07:05:53.51 UTC, :holder_id=>"cust_FtKx3V0QZXUg2cSCQMjfZ9CH", :object=>"voucher", :redemption=>{:quantity=>nil, :redeemed_quantity=>1, :object=>"list"}}}], :order=>{:id=>"ord_0f180c994ed1028c17", :source_id=>nil, :created_at=>2024-07-22 07:05:53.499 UTC, :updated_at=>nil, :status=>"PAID", :amount=>20000, :total_amount=>20000, :applied_discount_amount=>0, :total_applied_discount_amount=>0, :metadata=>{}, :customer_id=>nil, :referrer_id=>nil, :object=>"order", :redemptions=>{"r_0f180c999a51028c1e"=>{:date=>2024-07-22 07:05:53.513 UTC, :related_object_type=>"voucher", :related_object_id=>"v_B5u2zEUX75M0nGADR59zl5Ikd4qnxxeT", :related_object_parent_id=>"camp_1s1GPkPitiPpG6QJjK744cJR"}}}, :inapplicable_redeemables=>[], :skipped_redeemables=>[]}